ES6_Difference_between_Node_require_importAndexport.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 'ES6 Difference between Import and Export'
+slidebar_label: 'ES6 Difference between Import and Export'
+authors: [mohitmuktikant]
```

Comment:
The author `mohitmuktikant` is not defined. Please create a profile for the author by visiting the file [here](https://github.com/CodeHarborHub/codeharborhub.github.io/blob/e7d8ddcb499d1272cd69e02cc6c1c1eb787194f4/blog/authors.yml#L56) and adding the necessary information.